Package Is Not Compatible With net45 (.NetFramework,Version=v4.5).

VS 2017 ( Community ) Project is part of a multi-project solution.

Ideally everything would build against .net 4.7, but when I saw this message, I changed the offending project to build against 4.5.

Tried to add NuGet Package through VS 2017 NuGet Package Manager. No love.
Restarted VS. Tried again. No love.

Every time I get the same error message - 

Package Vlc.DotNet.Core.Interops 2.1.142 is not compatible with net45 (.NETFramework,Version=v4.5). Package Vlc.DotNet.Core.Interops 2.1.142 supports:
  - net20 (.NETFramework,Version=v2.0)
  - net35 (.NETFramework,Version=v3.5)
  - net40 (.NETFramework,Version=v4.0)
  - net45 (.NETFramework,Version=v4.5)
One or more packages are incompatible with .NETFramework,Version=v4.5.

Ideally this would work against .Net [Anything], but I'll just settle for it working HERE, preferably before I fling my computer through a wall.

I can reproduce this issue, like this:

project.cspoj:

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net45</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Vlc.DotNet.Core.Interops" Version="2.2.0-develop161" />
  </ItemGroup>
</Project>
```

And then:

```
dotnet restore
```

This gives me:

```
C:\Users\frede\demo\tests>dotnet restore
  Restoring packages for C:\Users\frede\demo\tests\LibVlcSample.csproj...
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202: Package Vlc.DotNet.Core.Interops 2.2.0-develop161 is not compatible with net45 (.NETFramework,Version=v4.5). Package Vlc.DotNet.Core.Interops 2.2.0-develop161 supports:\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net20 (.NETFramework,Version=v2.0)\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net35 (.NETFramework,Version=v3.5)\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net40 (.NETFramework,Version=v4.0)\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net45 (.NETFramework,Version=v4.5)
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202: Package Vlc.DotNet.Core.Interops 2.2.0-develop161 is not compatible with net45 (.NETFramework,Version=v4.5) / win7-x86. Package Vlc.DotNet.Core.Interops 2.2.0-develop161 supports:\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net20 (.NETFramework,Version=v2.0)\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net35 (.NETFramework,Version=v3.5)\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net40 (.NETFramework,Version=v4.0)\r
C:\Users\frede\demo\tests\LibVlcSample.csproj : error NU1202:   - net45 (.NETFramework,Version=v4.5)
  Generating MSBuild file C:\Users\frede\demo\tests\obj\LibVlcSample.csproj.nuget.g.props.
  Generating MSBuild file C:\Users\frede\demo\tests\obj\LibVlcSample.csproj.nuget.g.targets.
  Restore failed in 111.82 ms for C:\Users\frede\demo\tests\LibVlcSample.csproj.
```

--- a/nuget/Vlc.DotNet.Core.Interops.nuspec
+++ b/nuget/Vlc.DotNet.Core.Interops.nuspec
@@ -12,9 +12,9 @@
         <tags>vlc media wrapper interop interops</tags>
     </metadata>
     <files>
-        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 2.0\Vlc.DotNet.Core.Interops.dll" target="lib\net20\AnyCPU\Vlc.DotNet.Core.Interops.dll" />
-        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Core.Interops.dll" target="lib\net35\AnyCPU\Vlc.DotNet.Core.Interops.dll" />
-        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Core.Interops.dll" target="lib\net40\AnyCPU\Vlc.DotNet.Core.Interops.dll" />
-        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Core.Interops.dll" target="lib\net45\AnyCPU\Vlc.DotNet.Core.Interops.dll" />
+        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 2.0\Vlc.DotNet.Core.Interops.dll" target="lib\net20\Vlc.DotNet.Core.Interops.dll" />
+        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Core.Interops.dll" target="lib\net35\Vlc.DotNet.Core.Interops.dll" />
+        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Core.Interops.dll" target="lib\net40\Vlc.DotNet.Core.Interops.dll" />
+        <file src="..\build\Vlc.DotNet.Core.Interops\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Core.Interops.dll" target="lib\net45\Vlc.DotNet.Core.Interops.dll" />
     </files>
 </package>

--- a/nuget/Vlc.DotNet.Core.nuspec
+++ b/nuget/Vlc.DotNet.Core.nuspec
@@ -18,9 +18,9 @@
         </frameworkAssemblies>
     </metadata>
     <files>
-        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 2.0\Vlc.DotNet.Core.dll" target="lib\net20\AnyCPU\Vlc.DotNet.Core.dll" />
-        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Core.dll" target="lib\net35\AnyCPU\Vlc.DotNet.Core.dll" />
-        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Core.dll" target="lib\net40\AnyCPU\Vlc.DotNet.Core.dll" />
-        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Core.dll" target="lib\net45\AnyCPU\Vlc.DotNet.Core.dll" />
+        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 2.0\Vlc.DotNet.Core.dll" target="lib\net20\Vlc.DotNet.Core.dll" />
+        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Core.dll" target="lib\net35\Vlc.DotNet.Core.dll" />
+        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Core.dll" target="lib\net40\Vlc.DotNet.Core.dll" />
+        <file src="..\build\Vlc.DotNet.Core\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Core.dll" target="lib\net45\Vlc.DotNet.Core.dll" />
     </files>
 </package>

--- a/nuget/Vlc.DotNet.Forms.nuspec
+++ b/nuget/Vlc.DotNet.Forms.nuspec
@@ -21,9 +21,9 @@
         </frameworkAssemblies>
     </metadata>
     <files>
-        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 2.0\Vlc.DotNet.Forms.dll" target="lib\net20\AnyCPU\Vlc.DotNet.Forms.dll" />
-        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Forms.dll" target="lib\net35\AnyCPU\Vlc.DotNet.Forms.dll" />
-        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Forms.dll" target="lib\net40\AnyCPU\Vlc.DotNet.Forms.dll" />
-        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Forms.dll" target="lib\net45\AnyCPU\Vlc.DotNet.Forms.dll" />
+        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 2.0\Vlc.DotNet.Forms.dll" target="lib\net20\Vlc.DotNet.Forms.dll" />
+        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Forms.dll" target="lib\net35\Vlc.DotNet.Forms.dll" />
+        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Forms.dll" target="lib\net40\Vlc.DotNet.Forms.dll" />
+        <file src="..\build\Vlc.DotNet.Forms\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Forms.dll" target="lib\net45\Vlc.DotNet.Forms.dll" />
     </files>
 </package>

--- a/nuget/Vlc.DotNet.Wpf.nuspec
+++ b/nuget/Vlc.DotNet.Wpf.nuspec
@@ -22,8 +22,8 @@
        </frameworkAssemblies>
     </metadata>
     <files>
-        <file src="..\build\Vlc.DotNet.Wpf\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Wpf.dll" target="lib\net35\AnyCPU\Vlc.DotNet.Wpf.dll" />
-        <file src="..\build\Vlc.DotNet.Wpf\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Wpf.dll" target="lib\net40\AnyCPU\Vlc.DotNet.Wpf.dll" />
-        <file src="..\build\Vlc.DotNet.Wpf\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Wpf.dll" target="lib\net45\AnyCPU\Vlc.DotNet.Wpf.dll" />
+        <file src="..\build\Vlc.DotNet.Wpf\Release - AnyCPU - .Net 3.5\Vlc.DotNet.Wpf.dll" target="lib\net35\Vlc.DotNet.Wpf.dll" />
+        <file src="..\build\Vlc.DotNet.Wpf\Release - AnyCPU - .Net 4.0\Vlc.DotNet.Wpf.dll" target="lib\net40\Vlc.DotNet.Wpf.dll" />
+        <file src="..\build\Vlc.DotNet.Wpf\Release - AnyCPU - .Net 4.5\Vlc.DotNet.Wpf.dll" target="lib\net45\Vlc.DotNet.Wpf.dll" />
     </files>
 </package>

